### PR TITLE
doc(release): Prepare release notes for modules 22.04

### PR DESCRIPTION
**Pull request by pkippes**
Added RN for maps 22.04.11

## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [x] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [ ] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -168,6 +181,19 @@ The new MAP extension is now available in a full web version with a new server,
...
 
+### 22.04.11
+
+Release date: `September 22, 2023`
+
+#### Bug fixes
+
+-	[Desktop] Update dependency. 
+-	[Server] Fixed the diagnostic protocol test.
+
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md:192]: one missing (does not convey much infos though)
[Server] Fixed vulnarable property.

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -168,6 +181,19 @@ The new MAP extension is now available in a full web version with a new server,
...
 ## Centreon MAP Legacy
 
+### 22.04.11
+
+Release date: `September 22, 2023`
+
+#### Bug fixes
+
+-	[Desktop] Update dependency. 
+-	[Server] Fixed the diagnostic protocol test.
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md:191]: To remove as only for NG

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -15,6 +15,19 @@ Pour faire des demandes d'évolutions ou reporter des bugs sur les extensions co
...
 ## Centreon MAP
 
+### 22.04.11
+
+Release date: `September 22, 2023`
+
+#### Bug fixes
+
+-	[Desktop] Update dependency. 
+-	[Server] Fixed the diagnostic protocol test.
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md:25]: One missing 
[Editor] Fixed the deletion of source object when copy paste.

**Review by nmagniezCentreon — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -168,6 +181,19 @@ The new MAP extension is now available in a full web version with a new server,
...
+
+#### Enhancements
+
+- [Server] Makes links anchors consistent between viewer and editor.
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md:195]: To remove as only NG

**Review by nmagniezCentreon — CHANGES_REQUESTED** [review]

**Review by nmagniezCentreon — CHANGES_REQUESTED** [review]

**Review by nmagniezCentreon — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Pull request merged**